cdc/processor/pipeline/sink.go

```diff
@@ -136,10 +136,17 @@ func (n *sinkNode) flushSink(ctx pipeline.NodeContext, resolvedTs model.Ts) (err
 	if err := n.emitRow2Sink(ctx); err != nil {
 		return errors.Trace(err)
 	}
-	checkpointTs, err := n.sink.FlushRowChangedEvents(ctx, resolvedTs)
-	if err != nil {
-		return errors.Trace(err)
+
+	var checkpointTs uint64
+	var ok bool
+	// retry until ok is true
+	for !ok {
```

Comment:
why retry here?

Reply:
If not retry here, the `resolvedTs` pass in this `Flush` may be lost.